tests/integration/components/self-clear-cache-test.js

```diff
@@ -0,0 +1,93 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Unit | Component | self-clear-cache', (hooks) => {
+  setupRenderingTest(hooks);
+
+  test('should show last time user cleared the cache', async function (assert) {
+    assert.expect(1);
+
+    // set the outer context '23 March 1:23 pm IST'
```

Comment:
Nit. Can we remove the comment here? It just explains what the next line does. It's similar to saying

```
// Sets the variable called `a` to value 2
var a = 2'

```

Reply:
sure